src/main/java/core/basesyntax/FruitShopInfo.csv

```diff
@@ -0,0 +1,8 @@
+b,banana,20
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop#place-the-input-and-output-files-into-the-srcmainresources-folder)
**Place the input and output files into the src/main/resources folder.**